### PR TITLE
Implement historical fees in transaction details

### PR DESCRIPTION
**Pull request by amonsosanz**
This PR adds the historical fees information to the transaction details view.

### CODE REVIEW

**Review by adrifdez — APPROVED** [review]

File: src/views/transaction-details/transaction-details.view.tsx [hunk 1]
```diff
@@ -136,9 +199,29 @@ const TransactionDetails: FC = () => {
...
+            </Typography>
+            <Typography type="body1" className={classes.alignRow}>
+              {step2Fee} {env.tokens.ETH.symbol}
+            </Typography>
+          </div>
+        )}
         <div className={classes.row}>
           <Typography type="body2" className={classes.alignRow}>
-            Step 1/2
+            Step 1 Transaction
```
Comment (elias-garcia) [inline src/views/transaction-details/transaction-details.view.tsx:224]: I miss the word `Track` here as it was in the original designs from Chami. I would label this as: `Track step X transaction`

File: src/views/transaction-details/transaction-details.view.tsx [hunk 2]
```diff
@@ -12,20 +13,66 @@ import Chain from "src/views/transaction-details/components/chain/chain";
...
 import { formatTokenAmount } from "src/utils/amounts";
+import { Transaction, Token } from "src/domain";
+
+interface HistoricalFees {
+  step1?: string;
+  step2?: string;
+}
+
+const calculateTxFee = (txResponse: TransactionResponse, token: Token): string | undefined => {
+  if (txResponse.maxFeePerGas && txResponse.maxPriorityFeePerGas) {
```
Comment (elias-garcia) [inline src/views/transaction-details/transaction-details.view.tsx:29]: We already have this logic in the `estimateGasPrice` of the `BridgeContext`. Can we centralize this in a util that checks if the `FeeData` is from `EIP-1559` or not and just returns the fee?

File: src/views/transaction-details/transaction-details.view.tsx (hunk 1 above)
Comment (amonsosanz) [reply src/views/transaction-details/transaction-details.view.tsx]: Done

File: src/views/transaction-details/transaction-details.view.tsx (hunk 2 above)
Comment (amonsosanz) [reply src/views/transaction-details/transaction-details.view.tsx]: Done

**Review by elias-garcia — APPROVED** [review]

**Pull request merged**